### PR TITLE
Update scripts to support SLURM + pyxis/enroot environment

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
Signed-off-by: YuanTingHsieh <yuantinghsieh@gmail.com>

### Description
Update scripts to support SLURM + pyxis/enroot environment

### Status
**Ready**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by YuanTingHsieh** [comment]
Related to this issue: https://github.com/pytorch/ignite/issues/2048

File: acceleration/distributed_training/unet_evaluation_workflows.py [hunk 1]
```diff
@@ -22,11 +22,9 @@
   `--master_addr="192.168.1.1"`
   `--master_port=1234`
   For more details, refer to https://github.com/pytorch/pytorch/blob/master/torch/distributed/launch.py.
-  Alternatively, we can also use `torch.multiprocessing.spawn` to start program, but it that case, need to handle
```
Comment (Nic-Ma) [inline acceleration/distributed_training/unet_evaluation_workflows.py:25]: Thanks for the update.
Do you mean we can't use `torch.multiprocessing.spawn` anymore if using ignite?
CC @vfdev-5 .

Thanks.

File: acceleration/distributed_training/unet_evaluation_workflows.py [hunk 2]
```diff
@@ -22,11 +22,9 @@
...
-  all the above parameters and compute `rank` manually, then set to `init_process_group`, etc.
-  `torch.distributed.launch` is even more efficient than `torch.multiprocessing.spawn`.
 - Use `init_process_group` to initialize every process, every GPU runs in a separate process with unique rank.
   Here we use `NVIDIA NCCL` as the backend and must set `init_method="env://"` if use `torch.distributed.launch`.
+- We need the `--use_env` to avoid args.local_rank, ignite.distributed will handle that for us
```
Comment (Nic-Ma) [inline acceleration/distributed_training/unet_evaluation_workflows.py:27]: Missing a "." at end of the line.

File: acceleration/distributed_training/unet_evaluation_workflows.py (hunk 1 above)
Comment (vfdev-5) [reply acceleration/distributed_training/unet_evaluation_workflows.py:25]: @Nic-Ma I think it is unrelated to ignite. If you would like to replace torch distributed launch by `torch.multiprocessing.spawn`, user needs to update the code before running it as `python unet_evaluation_workflows.py --arg-num-proc-per-node=NUM_GPUS_PER_NODE --arg-nnodes=NUM_NODES --arg-node_rank=INDEX_CURRENT_NODE --arg-master-addr --arg-master-port` and also translate that into world size and rank. Ignite does that for the user like that: 
- spawn and launch unifiied approch: https://pytorch.org/ignite/distributed.html#ignite.distributed.launcher.Parallel please see examples below
- only spawning: https://pytorch.org/ignite/distributed.html#ignite.distributed.utils.spawn

Comment (Nic-Ma) [reply acceleration/distributed_training/unet_evaluation_workflows.py:25]: @vfdev-5 , thanks for your explanation!
@YuanTingHsieh , so I think you should not delete the previous doc-string about the `spawn` method.

Thanks.

Comment (vfdev-5) [reply acceleration/distributed_training/unet_evaluation_workflows.py:25]: A typo:
```diff
- but it that case
+ but in that case
```

**Comment by YuanTingHsieh** [comment]
Updated based on review comments

**Review by Nic-Ma — APPROVED** [review]
Thanks for the quick update!
It looks good to me now.

Thanks.

**Pull request merged**